stories/Checkbox.stories.mdx

```diff
@@ -0,0 +1,74 @@
+import { Meta, Canvas, Story, ArgsTable } from '@storybook/addon-docs';
+import { linkTo } from '@storybook/addon-links';
+import { Checkbox } from '../src'; // components you used in the story
+import {
+    StandardCheckbox,
+    CheckboxStates,
+    DisabledCheckbox,
+} from './examples/Checkbox';
+
+<Meta title="Form/Checkbox" component={Checkbox} />
+
+# Checkbox
+
+Checkboxes allow the user to select one or more items. They are used for filtering, selection, and toggles. They are typically used in forms, where the user can select one or more items from a set.
+> If you have a single option, avoid using a Checkbox and use a [Switch component](?path=/docs/form-switch--standard-switch) instead.
+
+## Table of Contents
+
+In this documentation we talk about the Checkbox component in depth about its usage and its props.
+
+- [Functionality](#functionality)
+    - [Checkbox States](#checkbox-states)
+    - [Disabled Checkbox](#disabled-checkbox)
+- [Other Props & Functionalities](#other-props--functionalities)
+
+## Functionality
+
+Functionally Checkbox has two states - checked and unchecked. Thanks to the `checked` prop, you can control
+the state of the Checkbox. It also comes with a `name` prop, that can be used to difrentiate between multiple Checkboxes.
```

Comment:
The `name` prop is used for group identity purposes. If you have a group of inputs, they all should have the same name. It's also required if used inside the Apollo `Form`